samigo/samigo-app/src/webapp/jsf/delivery/discrepancyInData.jsp

```diff
@@ -40,16 +40,15 @@
   <!-- content... -->
   <h3><h:outputText value="#{deliveryMessages.data_discrepancy_title}"/></h3>
   <f:verbatim>&nbsp;</f:verbatim>
+  <div class="alertMessage">
   <h:panelGrid border="0">
```

Comment:
Can you get rid of the use of tables entirely here? 

<h:panelGroup layout="block" id="data-disrepancy-container" styleClass="alertMessage">